src/main/java/ch/uzh/ifi/hase/soprafs22/controller/UserController.java

```diff
@@ -203,6 +204,21 @@ public void updateUserPreferences(
         userService.updatePreferences(user);
     }
 
+    @GetMapping("/users/{userId}/loginStatus")
+    @ResponseStatus(HttpStatus.OK)
+    @ResponseBody
+    public String getLoginStatus(
+      @RequestHeader Map<String, String> headers,
```

Comment:
Why do you Map here? we have other calls where w have a path variable and the header but none need Map..

Reply:
left over code from some experiment